### PR TITLE
ACM-3386 move lcl-clst ACM 2 mce

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
https://issues.redhat.com/browse/ACM-3386

Hi @ray-harris. Can you review the draft text for this issue in this PR? Thanks!

**Comment by cadawson** [comment]
@JakobGray: Will you take a look at this draft? Thanks!

**Review by JakobGray — CHANGES_REQUESTED** [review]
Now that hub-self-management is treated as a component of MCE I think the section can more closely resemble those below it, like the Hypershift add-on section. In that vein it could go something like this:

> By default, the cluster running {mce-short} will manage itself. To install with this feature disabled, include the following in the `MultiClusterEngine` values in `spec.overrides`:
(yaml content but set enabled to false)

Everything after "If the setting is not included in the YAML" can be removed, because that is describing the old way to do it by manually creating the resource. The last line "A hub cluster that is managed by itself..." can stay.

File: clusters/install_upgrade/adv_config_install.adoc [hunk 1]
```diff
@@ -6,34 +6,25 @@ The {mce-short} is installed using an operator that deploys all of the required
...
 
-. Run the following command to apply the file:
-+
-----
-oc apply -f import-hub.yaml
-----
+* The `name` value identifies the hub cluster as a `local-cluster`.
+* The `enabled` setting specifies whether the feature is enabled or disabled. When the value is `true`, the hub cluster manages itself. When the value is `false`, the hub cluster does not manage itself.
+
+If the setting is not included in the YAML file that defines the custom resource, you need to add it. The hub cluster can only be managed with this option.
```
Comment (JakobGray) [inline clusters/install_upgrade/adv_config_install.adoc]: This line: "If the setting is not included in the YAML file that defines the custom resource, you need to add it. The hub cluster can only be managed with this option." can be deleted, and then I think this is good to go.

Comment (cadawson) [reply clusters/install_upgrade/adv_config_install.adoc]: Made that update @JakobGray. Thanks! Does that need to be removed from the ACM version, as well, or is it ok there?

**Review by oafischer — APPROVED** [review]
lgtm

**Pull request merged**